packages/main/src/Tokenizer.ts

```diff
@@ -586,12 +596,12 @@ class Tokenizer extends UI5Element {
 		});
 
 		return this._getTokens().filter(token => {
-			const parentRect = this.contentDom.getBoundingClientRect();
+			const parentRect = this.contentDom!.getBoundingClientRect();
 			const tokenRect = token.getBoundingClientRect();
-			const tokenEnd = parseInt(tokenRect.right);
-			const parentEnd = parseInt(parentRect.right);
-			const tokenStart = parseInt(tokenRect.left);
-			const parentStart = parseInt(parentRect.left);
+			const tokenEnd = parseInt(String(tokenRect.right));
```

Comment:
do we need all the parsing? what is the return type of `tokenRect.right` ?

Reply:
right/left are numbers but parseInt is requires strings. if this is don't only for rounding maybe we can use another method like Math.round?

Reply:
@MapTo0 As Nayden has mentioned, right/left are numbers and parseInt requires string. 
I will just remove the parsing, as I think it is not needed.